README.md

```diff
@@ -1,81 +1,71 @@
-# ![moonbeam](media/moonbeam-cover.jpg)
+# ![Moonbeam](media/moonbeam-cover.jpg)
 
 ![Tests](https://github.com/PureStake/moonbeam/workflows/Release/badge.svg)
 
-Run an Ethereum compatible ~~parachain~~ (and blockchain for now, until parachains are more stable)
-based on Substrate.
+An Ethereum compatible ~~[Parachain](https://polkadot.network/technology/)~~ built with [Substrate](https://substrate.dev)
 
-_See [moonbeam.network](https://moonbeam.network) for the moonbeam blockchain description._  
-_See [www.substrate.io](https://www.substrate.io/) for substrate information._
 
-## Install (linux)
+_Discover the Moonbeam project at [moonbeam.network](https://moonbeam.network)._
+_Learn to [use the Moonbeam network](https://docs.moonbeam.network/) with our technical docs._
+_Reference our [crate-level docs (rustdocs)](https://purestake.github.io) to contribute._
 
-### Get the code
+## Run a node with Docker
 
-Get the tutorial specific tag of the PureStake/Moonbeam repo:
+Docker images are published for every tagged release. Learn more with `moonbeam --help`.
 
 ```bash
-git clone -b tutorial-v3 https://github.com/PureStake/moonbeam
-cd moonbeam
+# Join the public testnet
+docker run --network="host" purestake/moonbeam:v0.6.1 --chain alphanet
 ```
 
-### Setting up enviroment
+### Local development node
 
-Install Substrate pre-requisites (including Rust):
+Developers who are building dApps to run on moonbeam, may want a lightweight node to work with
+locally. You can quickly spin up a single node with no relay chain backing it using the development
+service.
 
 ```bash
-curl https://getsubstrate.io -sSf | bash -s -- --fast
+# Run a dev service node.
+docker run --network="host" purestake/moonbeam:v0.6.1 --dev
 ```
 
-Run the initialization script, which checks the correct rust nightly version and adds the
-`wasm32-unknown-unknown` target to that specific version:
-
-```bash
-./scripts/init.sh
-```
 
 ## Build the Moonbeam Node
 
-Build the corresponding binary file:
+To build Moonbeam, you will need a proper Substrate development environment. If you've never worked
+with Substrate before, check out https://substrate.dev/docs/en/knowledgebase/getting-started/
 
 ```bash
+# Fetch the code
+git clone https://github.com/PureStake/moonbeam
+cd moonbeam
+
+# Build the node (The first build will be long (~30min))
```

Comment:
Also it would be nice to include the instructions for the one liner to get rust (or at least information on where to get it)

Reply:
In the text above this snippet I wrote that you need a Substrate development environment and linked more info about that. That resource points to the one liner and explains the manual way for various platforms as well. My goal was that we wouldn't duplicate that content in the readme.

Actually, your docs cover that well too https://docs.moonbeam.network/getting-started/local-node/setting-up-a-node/ I'll link to them again in this section.